src/webob/acceptparse.py

```diff
@@ -555,10 +592,10 @@ def header_value(self):
     @property
     def parsed(self):
         """
-        (``list`` or ``None``) Parsed form of the header.
+        (``tuple`` or ``None``) Parsed form of the header.
 
-        A list of (*media_range*, *qvalue*, *media_type_params*,
-        *extension_params*) tuples, where
+        A tuple of ``(*media_range*, *qvalue*, *media_type_params*,
+        *extension_params*)`` tuples, where
```

Comment:
Same issue as https://github.com/Pylons/webob/pull/460/files#r1531436169 with the asterisks here and in the rest of this docstring.